docs/integrations/code-conventions.md

```diff
@@ -711,7 +738,99 @@ tableToMarkdown('Data Table', d, headers=('id', 'url1', 'result', 'links'),
 The resulted table will be:
 
 ![image](https://user-images.githubusercontent.com/72340690/103922604-a25efb80-511c-11eb-9021-c062226b5001.png)
+#
+Use the ```date_fields``` argument (list) of date fields to format the value to human-readable output.
+```python
+data = [
+    {
+        "docker_image": "demisto/python3",
+        "create_time": '1631521313466'
+    }
+]
+tableToMarkdown('tableToMarkdown test', data, headers=["docker_image", "create_time"],
+                date_fields=['create_time'])
+```
+The above will create the table seen below:
+### tableToMarkdown test
+|docker_image|create_time|
```

Comment:
```suggestion
tableToMarkdown('tableToMarkdown date_fields example', data, headers=["docker_image", "create_time"],
                date_fields=['create_time'])
```
The above will create the table seen below:
### tableToMarkdown date_fields example
|docker_image|create_time|
```